src/layout/Tabs/Tabs.tsx

```diff
@@ -104,13 +102,13 @@ function TabHeader({
       {!!icon && (
         <img
           src={icon}
-          alt={translatedTitle ?? 'tab icon'}
+          alt=''
```

Comment:
This would break WCAG requirements, I think..? :thinking: You can still use `langAsString()` here even if you use `<Lang />` below.

Reply:
No actually it wont. The designsystem is handling it. It takes the tab text an relates it to the icon.  You get a warning when you add alt text, saying that it is not necessary to have multiple text describing the image. But you should not remove the alt tag, so I added an empty one(It is the wcag way)